chain/chain/src/test_utils/kv_runtime.rs

```diff
@@ -388,13 +388,8 @@ impl KeyValueRuntime {
         Ok(None)
     }
 
-    fn get_congestion_info(protocol_version: ProtocolVersion) -> Option<CongestionInfo> {
-        if ProtocolFeature::CongestionControl.enabled(protocol_version) {
-            // TODO(congestion_control) - properly initialize
-            Some(CongestionInfo::default())
-        } else {
-            None
-        }
+    fn get_congestion_info(_protocol_version: ProtocolVersion) -> Option<CongestionInfo> {
```

Comment:
Change function to not take `_protocol_version` as parameter and return `CongestionInfo` instead of `Option<CongestionInfo>`